packages/components/src/toggle-group-control/stories/index.js

```diff
@@ -71,6 +71,11 @@ const _default = ( { options } ) => {
 				opt[ 'aria-label' ],
 				`${ KNOBS_GROUPS.ToggleGroupControlOption }-${ index + 1 }`
 			) }
+			tooltipText={ text(
+				`${ KNOBS_GROUPS.ToggleGroupControlOption }: tooltipText`,
+				opt.tooltipText,
+				`${ KNOBS_GROUPS.ToggleGroupControlOption }-${ index + 1 }`
+			) }
```

Comment:
I think this was missed in https://github.com/WordPress/gutenberg/pull/36726/commits/761dacf5a0aa8598bc3138d71dd3de3b5e299025 when switching to use `showTooltip` instead.

```suggestion
			showTooltip={ boolean(
				`${ KNOBS_GROUPS.ToggleGroupControlOption }: showTooltip`,
				opt.showTooltip,
				`${ KNOBS_GROUPS.ToggleGroupControlOption }-${ index + 1 }`
			) }
```

Reply:
Thanks for catching that. Yes, I complete forgot to update the story!